test/integration/video/uploads.spec.js

```diff
@@ -0,0 +1,58 @@
+require('dotenv').config();
+const { expect } = require('chai');
+const Mux = require('../../../src/mux');
+
+/** @test {Uploads} */
+describe('Integration::Uploads', () => {
+  const muxClient = new Mux();
+  const { Video } = muxClient;
+  let testUpload;
+  const createdUploads = []; // These are uploads we'll clean up when it's all done.
+
+
+  before(async () => {
```

Comment:
Why do you do this? You have a test specifically for this, and then create one directly in each test where you need to do things...

Reply:
Except for the get call... why not just create it in the get test?